string name
sphero/SpheroCollision data